frontend/css/layout.css

```diff
@@ -0,0 +1,124 @@
+
+.container {
+   display: flex;
+}
+
+.grid {
+   display: grid;
+}
+
+a {
+   text-decoration: none;
+   color: inherit;
+}
+/* Account management */
+
+.account {
+   justify-content: flex-end;
+   padding-right: 20px;
```

Comment:
- Where we're currently using px, change to rems. 
- Add the spacing values as css variables in for example `spacings.css`

Reply:
Done in #47